src/Datagrid/DatagridInterface.php

```diff
@@ -104,4 +108,11 @@ public function hasActiveFilters();
      * @return bool
      */
     public function hasDisplayableFilters();
+
+    /*
+     * NEXT_MAJOR: Uncomment getSortParameters and getPaginationParameters.
```

Comment:
We don't need them when there are `@method` methods

Reply:
I assume that @method makes a soft-check of the implementation when adding the method to the interface makes a hard-check of the implementation, leading to a php error if it's not implemented. 
Since we'll need the method, it will be needed.

Plus, for consistency, I think it's better to have all the method in the interface and not half of them declare with the `@method` tag.

Reply:
I mean we must check all `@method` methods when doing a major release and add them to the interface.

AFAIR I removed some `NEXT_MAJOR`s when adding these `@method`s on the interfaces

Reply:
There is still `NEXT_MAJOR` comment for `@method` in `interface AdminExtensionInterface` and I saw lot of others.

I agree we *must* check all `@method`. But `NEXT_MAJOR` comment seems to be the best way to not forget to do something.